### ISSUE TITLE
Inference fails.

### ISSUE THREAD

**Issue by pranoyr** [lucidrains/parti-pytorch#9]
Training goes good. 

During training , to see if the network is giving plausible images, i visualised the logits and gave it to vq decoder which decoded into good images.

But during autoregressive inference, the network is not giving good images conditioned on text.  

Between , I use a different VitVQGAN( not from this repo).
